core/src/main/resources/messages/messages.properties

```diff
@@ -98,6 +100,8 @@ error.goal.update.time.before.original=Creation time of updated goal ({0}) is be
 error.no.device.request.present=No new device request present for mobile number ''{0}''
 error.device.request.expired=New device request expired for mobile number ''{0}''
 error.device.request.invalid.password=Invalid temporary password to use new device request for mobile number ''{0}''. Maybe the request was replaced with a different secret?
+error.device.unknown.operating.system=Operating system ''{0}'' is not supported.
+error.device.invalid.device.name=Device name ``{0}`` is invalid. A valid device name is not longer than {1} characters and does not contain a ''{2}''.
```

Comment:
Wrong quotes (`` instead of '')

Reply:
There were a few more of these. Corrected them all.